### DISCUSSION TITLE
How can I use create-next-app with a template and NOT use TypeScript?

### DISCUSSION THREAD

**Discussion by ianyoung** [vercel/next.js#46491] [Help]
It seems that when using a template you don't get asked whether you want to use TypeScript or not:

```
npx create-next-app --example with-mongodb nextjs-test
Need to install the following packages:
  create-next-app@13.2.1
Ok to proceed? (y) y
Creating a new Next.js app in ...
```
How can I use a template without TypeScript?

**Comment by icyJoseph** [comment]
Sorry mate, but most examples were migrated to TypeScript. 

I saw an issue opened by someone who was not pleased by this either.

There's a workaround, which is to find an older version of the template, but that's not ideal either.

- https://github.com/vercel/next.js/issues/46100
